### ISSUE TITLE
Jupyter widgets supports binary data in messages

### ISSUE THREAD

**Issue by vidartf** [InsightSoftwareConsortium/itk-jupyter-widgets#29]
Hi! I see that this repo uses base64 data a lot, so I thought I should point this out: While it is a poorly documented feature, any binary data in the serialized state of a widget will be handled by ipywidgets. The binary types are defined as follows:

```python
if PY3:
    _binary_types = (memoryview, bytearray, bytes)
else:
    _binary_types = (memoryview, bytearray)
```

I hope this can be useful to you!

**Comment by vidartf** [comment]
PS: This feature is new from ipywidgets 7, if I remember correctly.

**Comment by thewtex** [comment]
Thanks for the tip, @vidartf ! I will create a patch...

**Comment by thewtex** [comment]
Addressed in #31
